modules/vf-graphql-holochain/mutations/agreement.ts

```diff
@@ -5,8 +5,8 @@
  * @since:   2020-06-19
  */
 
-import { ByRevision, DNAIdMappings } from '../types'
-import { mapZomeFn } from '../connection'
+import { ByRevision, DNAIdMappings } from '../types.js'
```

Comment:
curious why you've added these... I shouldn't think adding extensions would be necessary?

Reply:
I don't know why they were, but they were. Perhaps there is a configuration via which we could not use them.

Reply:
But they're not, are they? There are still `type` & `interface` defs as well as function generics in these source files. I want to be sure we're not breaking type inference for Typescript modules dependent on these...